src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main>
+      <div class="calendar calendar--start_day-sat calendar__month_length--29 calendar--container">
```

Comment:
```suggestion
      <div class="calendar calendar--start-day--sat calendar--month-length--29">
```
Check your BEM syntax

Reply:
Also, why do you need `calendar--container`?